Package path ./dist/types/sessions is not exported from @abstract-foundation/agw-client

I am trying to use the **@abstract-foundation/agw-react** `^0.1.3` and I am getting this error.

```
ERROR in ../node_modules/@abstract-foundation/agw-react/dist/esm/hooks/useRevokeSessions.js 2:0-85
Module not found: Error: Package path ./dist/types/sessions is not exported from package /Users/***/Repos/giga-os/node_modules/@abstract-foundation/agw-client (see exports field in /Users/***/Repos/giga-os/node_modules/@abstract-foundation/agw-client/package.json)
```
some of my dependencies

```package.json
  "devDependencies": {
    "@babel/core": "^7.26.0",
    "@babel/preset-env": "^7.26.0",
    "@babel/preset-react": "^7.25.9",
    "@babel/preset-typescript": "^7.26.0",
    "@types/react": "^18.3.12",
    "@types/react-dom": "^18.3.1",
    "@types/webpack": "^5.28.5",
    "babel-loader": "^9.2.1",
    "typescript": "^5.6.3",
    "webpack": "^5.95.0",
  },
  "dependencies": {
    "@abstract-foundation/agw-client": "^0.1.2",
    "@abstract-foundation/agw-react": "^0.1.3",
    "@privy-io/cross-app-connect": "^0.1.2",
    "@privy-io/react-auth": "^1.98.4",
    "@tanstack/react-query": "^5.62.11",
    "ethers": "^6.13.4",
    "react": "^18.3.1",
    "react-dom": "^18.3.1",
    "secp256k1": "^5.0.1",
    "siwe": "^2.3.2",
    "viem": "^2.21.37",
    "wagmi": "^2.14.6"
  },
```

My usage is fairly simple and could be sumarised as:

```typescript
  const { login, logout } = useLoginWithAbstract();
  const { data: abstractClient, error, isLoading } = useAbstractClient();

  if (isLoading) return <div>Loading Abstract Client...</div>;
  if (error) return <div>Error: {error.message}</div>;

  return (
    <div>
      <button onClick={login}>Connect with Abstract</button>
      <button onClick={logout}>Disconnect</button>
    </div>
  )
```
After checking the useRevokeSessions.js I found it is in importing the getSessionHash type.

```typescript
import { getSessionHash } from '@abstract-foundation/agw-client/dist/types/sessions';`
```

But after checking the **package.json** and the exported types files`./dist/{ esm | cjs | types }/exports/index.{js | .d.ts}`, neither exports the required session type and the build. Honestly not sure if I am missing something or there is a bug.

--- a/.changeset/fast-donkeys-join.md
+++ b/.changeset/fast-donkeys-join.md
@@ -1,0 +1,6 @@
+---
+'@abstract-foundation/agw-client': patch
+'@abstract-foundation/agw-react': patch
+---
+
+Fix incorrect package import from sessions

--- a/packages/agw-client/src/exports/sessions.ts
+++ b/packages/agw-client/src/exports/sessions.ts
@@ -15,6 +15,7 @@ import type {
 } from '../sessions.js';
 import {
   ConstraintCondition,
+  getSessionHash,
   LimitType,
   LimitUnlimited,
   LimitZero,
@@ -25,6 +26,7 @@ export {
   type Constraint,
   ConstraintCondition,
   createSessionClient,
+  getSessionHash,
   type Limit,
   LimitType,
   LimitUnlimited,

--- a/packages/agw-react/src/hooks/useRevokeSessions.ts
+++ b/packages/agw-react/src/hooks/useRevokeSessions.ts
@@ -1,6 +1,6 @@
 import { sessionKeyValidatorAddress } from '@abstract-foundation/agw-client/constants';
-import { getSessionHash } from '@abstract-foundation/agw-client/dist/types/sessions';
 import {
+  getSessionHash,
   type SessionConfig,
   SessionKeyValidatorAbi,
 } from '@abstract-foundation/agw-client/sessions';

